reconcile/utils/saasherder/saasherder.py

```diff
@@ -742,7 +743,7 @@ def get_archive_info(
         [url, sha] = trigger_reason.split(" ")[0].split("/commit/")
         repo_name = urlparse(url).path.strip("/")
         file_name = f"{repo_name.replace('/', '-')}-{sha}.tar.gz"
-        if "github" in url:
+        if url.startswith(GITHUB_BASE_URL):
```

Comment:
Can you refactor this pattern into a utils method?

```
if is_github(url):
...
```

Reply:
I'll extract something in the next step `need to spike if github repos can be done in similar way.`. And combine it with repo name extraction, like this:

```python
VCS.parse_url(url)
```

the result is like

```
platform: "github" | "gitlab"
repo: "abc/def"
```

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4994